mopidy_mpd/network.py

```diff
@@ -14,6 +14,14 @@
 
 CONTROL_CHARS = dict.fromkeys(range(32))
 
+def get_systemd_socket():
+    """Attempt to get a socket from systemd."""
+    try:
+        fdnames = os.environ.get("LISTEN_FDNAMES", "").split(":")
+        fd = fdnames.index("mpd") + 3  # 3 is the first systemd file handle
+        return socket.socket(fileno=fd)
+    except ValueError:
+        return None
```

Comment:
I assume the `ValueError` is from `fdnames.index("mpd")` when `"mpd"` is not one of the `fdnames`, and not from something else as well?

If so, I'd much rather be explicit about this and avoid the quite generic except clause:

```suggestion
    fdnames = os.environ.get("LISTEN_FDNAMES", "").split(":")
    if "mpd" not in fdnames:
        return None
    fd = fdnames.index("mpd") + 3  # 3 is the first systemd file handle
    return socket.socket(fileno=fd)
```